cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -45,11 +45,31 @@ const (
 	pluginMountPath      = "/var/lib/kubelet/device-plugins"
 	kubeletEndpoint      = "kubelet.sock"
 	pluginEndpointPrefix = "sriovNet"
-	resourceName         = "intel.com/sriov"
+	resourceName         = "netdev/sriov"
 )
 
+// default NIC Model to be discovered
+var defaultNicModel = []string{"0x8086-0x10fb", "0x8086-0x154c"}
```

Comment:
I am not sure why we need a default card here. It think it will be better to let to user to configure what he want to use and if it empty is should take all cards. What do you think?

Reply:
Agree, I think we want a general device plugin that works with various NIC models and not tied to certain NIC types, having a default list may cause confusion and require frequent change when adding more NIC models. Will submit a new version of commit to fix it.

Reply:
Fixed with a follow-up commit, defaults to discover all capable & configured SR-IOV devices when ``nic-model`` is not specified.